Formula/benerator.rb

```diff
@@ -0,0 +1,97 @@
+class Benerator < Formula
+  desc "Tool for realistic test data generation"
+  homepage "https://rapiddweller.github.io/homebrew-benerator/"
+  url "https://github.com/rapiddweller/rapiddweller-benerator-ce/releases/download/3.1.0/rapiddweller-benerator-ce-3.1.0-jdk-11-dist.tar.gz"
+  sha256 "194feb051ae18cfcd407b8e1668ce9c60561394bc454f9fc9747c274166843bc"
+
+  license "Apache-2.0"
+  depends_on "openjdk"
+
+  def install
+    # Remove unnecessary files
+    rm_f Dir["bin/*.bat", "bin/pom.xml"]
+
+    # Installs only the "bin" and "lib" directories from the tarball
+    libexec.install Dir["bin", "lib"]
+    # Generate a script that sets the necessary environment variables
+    (bin/"benerator").write_env_script(
+      libexec / "bin/benerator",
+      JAVA_HOME:      Language::Java.java_home,
+      BENERATOR_HOME: libexec,
+    )
+  end
+
+  test do
+    # Test if version is correct
+    assert_match "Benerator Community Edition #{version}-jdk-11",
+                 shell_output("#{bin}/benerator --version")
+    assert_includes shell_output("#{bin}/benerator --version"), "Java version:  #{Formula["openjdk"].version}"
```

Comment:
That's not the expected order:
```suggestion
    assert_match "Java version:  #{Formula["openjdk"].version}", shell_output("#{bin}/benerator --version")
```

Reply:
adjusted